rpm/modules-static-context.patch

```diff
@@ -0,0 +1,30 @@
+--- modules.yaml	2021-05-10 10:28:46.822338342 -0400
++++ static_context-modules.yaml	2021-05-10 10:50:50.110402154 -0400
+@@ -5,7 +5,8 @@
+   name: kangaroo
+   stream: 0
+   version: 20180730223407
+-  context: deadbeef
++  static_context: true
++  context: 0123456789
+   arch: noarch
+   summary: Kangaroo 0.3 module
+   description: >-
+@@ -28,7 +29,8 @@
+   name: kangaroo
+   stream: 0
+   version: 20180704111719
+-  context: deadbeef
++  static_context: true
++  context: 0123456788
+   arch: noarch
+   summary: Kangaroo 0.2 module
+   description: >-
+@@ -51,6 +53,7 @@
+   name: duck
+   stream: 0
+   version: 201809302113907
++  static_context: false
+   context: deadbeef
+   arch: noarch
+   summary: Duck 0.8 module
```

Comment:
The modules to set static_context for, were they chosen with the dep solving testing in mind?
It's a pain to change fixtures, because it often will require adjusting tests which use it.
I think we either need to think what would be the best set of modules to have static_context set to test dep solving properly when we add this feature, or we will need a new fixture for dep solving when we get to it. And it's ok, just saying it out loud to be sure we are on the same page.